pkg/install/resources.go

```diff
@@ -293,31 +293,31 @@ func AllResources(o *VeleroOptions) *unstructured.UnstructuredList {
 	secretPresent := o.SecretData != nil
 
 	deployOpts := []podTemplateOption{
-		WithAnnotations(o.PodAnnotations),
-		WithLabels(o.PodLabels),
-		WithImage(o.Image),
-		WithResources(o.VeleroPodResources),
-		WithSecret(secretPresent),
-		WithDefaultRepoMaintenanceFrequency(o.DefaultRepoMaintenanceFrequency),
-		WithServiceAccountName(serviceAccountName),
-		WithGarbageCollectionFrequency(o.GarbageCollectionFrequency),
-		WithUploaderType(o.UploaderType),
```

Comment:
We [use these functions](https://github.com/openshift/oadp-operator/blob/94cf689eee35e51d34804393f6291aca21d5a52a/controllers/velero.go#L311-L320) in openshift/oadp-operator. Any way to keep these exported?

Reply:
Sure. I will make some change to ignore this linter warning.

Reply:
unexported-return: exported func WithFeatures returns unexported type install.podTemplateOption, which can be annoying to use (revive)